source/localizable/tutorial/installing-addons.md

```diff
@@ -0,0 +1,85 @@
+Ember has a rich ecosystem of addons that can be easily added to projects.
+Addons provide a wide range of functionality to projects, often saving time and letting you focus on your project's value.
+
+To browse addons, visit the [Ember Observer](https://emberobserver.com/) website.  It catalogs and categorizes ember addons that have been published to NPM and assigns them a score based on a variety of criteria.
+
+For Super Rentals, we'll take advantage of two addons: [ember-cli-tutorial-style](https://github.com/toddjordan/ember-cli-tutorial-style) and [ember-cli-mirage](http://www.ember-cli-mirage.com/).
+
+### ember-cli-tutorial-style
+
+Instead of having you copy/paste in CSS to style Super Rentals, we've created an addon called [ember-cli-tutorial-style](https://github.com/toddjordan/ember-cli-tutorial-style) that instantly adds CSS to the tutorial.
+The addon works by creating a file called `ember-tutorial.css` and putting that file in the application's `/vendor` directory.
+At compile time the contents of the css file is automatically concatenated to the `vendor.css` file which is imported into the application's `apps/index.html`.
+By creating the file in the `vendor` directory, the addon is exposing it to the developer for potential customization (in case we want to tweak the styles).
+
+Run the following command to install the addon:
+
+```shell
+ember install ember-cli-tutorial-style
+```
+
+Starting the server will incorporate the new css and refreshing the screen will give you this:
+
+![super rentals styled homepage](../../images/installing-addons/styled-super-rentals-basic.png)
+
+### ember-cli-mirage
+
+[Mirage](http://www.ember-cli-mirage.com/) is a client HTTP stubbing library often used for Ember acceptance testing.
+For the case of this tutorial, we'll use mirage as our data source.
+Mirage will allow us to create fake data to work with while developing our app and mimic a running backend server.
+
+Install the Mirage addon as follows:
+
+```shell
+ember install ember-cli-mirage
+```
+
+If you were running `ember serve` in another shell, restart the server to include Mirage in your build.
+
+Let's now configure Mirage to send back our rentals that we had defined above by updating `app/mirage/config.js`:
```

Comment:
Sidenote: Mirage is moving all files to `/mirage/` in v0.2.0, so we'll need to update the tutorial when v0.2.0 releases